addons/website_sale_slides/tests/test_course_purchase_flow.py

```diff
@@ -74,3 +74,51 @@ def test_course_purchase_flow(self):
 
         self.assertIn(self.user_portal.partner_id, self.channel.partner_ids)
         self.assertIn(self.user_portal.partner_id, self.channel_2.partner_ids)
+
+    def test_course_product_synchro(self):
+        """ Test the synchronization between a course and its product """
+        course_1 = self.env['slide.channel'].with_user(self.user_publisher).create({
```

Comment:
Could use a `@users` to set "user_publisher" for the whole test.

That way we test the flow with a real user.